### ISSUE TITLE
Jackett Pull - Cardigann - Size Parsing

### ISSUE THREAD

**Issue by bakerboy448** [Prowlarr/Prowlarr#485]
Review if https://github.com/Jackett/Jackett/commit/58c4f75f70f70d52f304980a74b84135cd47a482 is needed

Note: Current PR includes the size changes
https://github.com/Prowlarr/Indexers/pull/36

**Comment by bakerboy448** [comment]
also review https://github.com/Jackett/Jackett/commit/2d3e079eb6a3c602864af3872aa87fe6a8c7c430

**Comment by bakerboy448** [comment]
looks like https://github.com/Prowlarr/Prowlarr/blob/e304461dfb242544e741be81253572a82b9e6632/src/NzbDrone.Core/Parser/Model/ReleaseInfo.cs#L103-108  is may be okay for the size change? (https://github.com/Jackett/Jackett/commit/58c4f75f70f70d52f304980a74b84135cd47a482)

and seems we may want to pull https://github.com/Jackett/Jackett/commit/2d3e079eb6a3c602864af3872aa87fe6a8c7c430 for 
https://github.com/Prowlarr/Prowlarr/blob/e304461dfb242544e741be81253572a82b9e6632/src/NzbDrone.Core/Indexers/Definitions/Cardigann/CardigannParser.cs#L216

@Qstick and @ta264 initial thoughts?

**Comment by bakerboy448** [comment]
review if https://github.com/jackett/jackett/commit/2ea2b0b1def70fee1aa8550751bf3882a7ff9eab is needed

**Comment by bakerboy448** [comment]
review if https://github.com/Jackett/Jackett/commit/2ea2b0b1def70fee1aa8550751bf3882a7ff9eab is needed

**Comment by garfield69** [comment]
be sure to also pull https://github.com/Jackett/Jackett/commit/f2c760ce8c278f07d732ef9efefdf3e8c0d77473 which fixes bug in https://github.com/Jackett/Jackett/commit/58c4f75f70f70d52f304980a74b84135cd47a482

**Comment by Qstick** [comment]
Thanks @garfield69! Why use GetBytes for the int vales like seeds, grabs, and such, feels a bit cleaner to fixup NormalizeNumber method in ParseUtil doesn't it?

**Comment by garfield69** [comment]
I felt that rather than have two (or more) places where we were dealing with  numbers that had commas, dots and dashes, that the one routine that does it all seemed better to me.

**Comment by Qstick** [comment]
> I felt that rather than have two (or more) places where we were dealing with  numbers that had commas, dots and dashes, that the one routine that does it all seemed better to me.

Right, Good thinking though we should likely change the name in that case, or consolidate and streamline our number handling methods

**Comment by bakerboy448** [comment]
need to add 58206c3ba753feb6af86428551eb0fcbcdc1555d

**Comment by bakerboy448** [comment]
need to add https://github.com/Jackett/Jackett/commit/bdff2209bd80fcf388200589c691d9885f1e9a39

**Comment by bakerboy448** [comment]
for posterity:
> outstanding jackett pulls are a bit over my head with how the logic between jackett and us has diverged.
> 
> (at)ta264 (at)Qstick likely will need some of ya'lls help finishing this off with the 3 outstanding commits to pull
> 
> along with any changes Q wants around the getbytes function
> 
> also this is brewing
> [Jackett/Jackett#12295](https://github.com/Jackett/Jackett/pull/12295)
https://github.com/Prowlarr/Prowlarr/pull/491#issuecomment-918426096

**Comment by bakerboy448** [comment]
- [ ] add https://github.com/Jackett/Jackett/commit/4c418ed4eb8740b6fe69975beb66bde696911711

**Comment by bakerboy448** [comment]
- [ ] add https://github.com/Jackett/Jackett/commit/5846e9b4478d995fe45549ffca4913c4d2c3fc5b

**Comment by bakerboy448** [comment]
- [ ] add https://github.com/Jackett/Jackett/commit/3ba608176643c9fbd3c37c0503a5cd073e485771

**Comment by bakerboy448** [comment]
- [ ] add https://github.com/Jackett/Jackett/commit/bd8c0bfafee31845b9f7f1b4edcd3470675e6b68

**Comment by bakerboy448** [comment]
Thread cleansed and OP updated with consolidated outstanding commits
